src/components/redaction/redaction-media-library-tab.js

```diff
@@ -165,7 +167,7 @@ function RedactionMediaLibraryTab({ mediaLibraryItems, onMediaLibraryItemsChange
             </div>
           </div>
         }
-        onTitleClick={() => handleInfoCellTitleClick(row)}
+        onTitleClick={event => handleInfoCellTitleClick(row, event)}
```

Comment:
Very cool, now could you also do the same for the other 2 Redaction tabs and the Search page? 😋 
The ticket at least mentions the Search page only, though I do agree that it's nice to have this consistency on both these pages.

Reply:
Scratch all that and thanks for clarifying!